scripts/variantstore/wdl/GvsCallsetCost.wdl

```diff
@@ -2,11 +2,11 @@ version 1.0
 
 workflow GvsCallsetCost {
     input {
-#        String project_id
-#        String dataset_name
+        String project_id
+        String dataset_name
         String workspace_namespace
         String workspace_name
-#        String callset_name
+        String callset_identifier
```

Comment:
nit: change to call_set_identifier to be consistent with table field?

Reply:
yeah this was bugging me too... I was wondering whether to standardize on `call_set` or `callset`, sounds like you prefer the former.